conans/client/generators/cmake_find_package.py

```diff
@@ -60,14 +61,18 @@ class CMakeFindPackageGenerator(Generator):
 
         include(FindPackageHandleStandardArgs)
 
-        conan_message(STATUS "Conan: Using autogenerated Find{{ pkg_name }}.cmake")
-        set({{ pkg_name }}_FOUND 1)
+        conan_message(STATUS "Conan: Using autogenerated Find{{ pkg_filename }}.cmake")
+        set({{ pkg_filename }}_FOUND 1)
+        {% set pkg_filename_upper = pkg_filename | upper %}
+        {% if pkg_filename == pkg_filename_upper %}
+        set({ pkg_filename_upper }_FOUND 1)
+        {% endif %}
```

Comment:
Probably defining it twice doesn't harm, so just avoid adding more templating logic
```suggestion
        set({  pkg_filename | upper }_FOUND 1)
```